src/config/locales/fr/translation.json

```diff
@@ -0,0 +1,237 @@
+      "confirm_message": "Êtes-vous certain de vouloir supprimer toutes les associations ?"
+    },
+    "language": {
+      "title": "Langue",
+      "display_language": "Langue",
+      "notice": "Nécessite un redémarrage",
+      "confirm": {
+        "title": "Avertissement",
+        "message": "Vous devez redémarrer l'application pour appliquer les modifications, continuer ?",
+        "ok": "Redémarrer",
+        "cancel": "Annuler"
+      }
+    }
+  },
+  "modals": {
+    "new_toot": {
+      "title": "Nouveau pouet",
+      "cw": "Écrivez votre avertissement ici",
+      "status": "Qu'avez-vous en tête ?",
+      "cancel": "Annuler",
+      "toot": "Pouet!"
+    },
+    "jump": {
+      "jump_to": "Aller à..."
+    },
+    "add_list_member": {
+      "title": "Ajouter un membre à la liste",
+      "account_name": "Nom du compte"
+    },
+    "list_membership": {
+      "title": "Éditer la liste"
+    }
+  },
+  "cards": {
+    "toot": {
+      "show_more": "Voir plus",
+      "hide": "Cacher",
+      "sensitive": "Afficher le contenu sensible",
+      "view_toot_detail": "Voir les détails du pouet",
+      "open_in_browser": "Ouvrir dans un navigateur",
+      "copy_link_to_toot": "Copier le lien du pouet",
+      "delete": "Effacer",
+      "via": "par {{application}}"
+    }
+  },
+  "side_bar": {
+    "account_profile": {
+      "follows_you": "Abonné⋅e",
+      "doesnt_follow_you": "Pas abonné⋅e",
+      "open_in_browser": "Ouvrir dans un navigateur",
+      "manage_list_memberships": "Inscriptions aux listes",
+      "toots": "Pouets",
+      "follows": "Abonnements",
+      "followers": "Abonné⋅e⋅s"
+    }
+  },
+  "hashtag": {
+    "tag_name": "Nom du hashtag"
+  },
+  "search": {
+    "search": "Rechercher",
+    "account": "Compte",
+    "keyword": "mot clé"
+  },
+  "lists": {
+    "index": {
+      "new_list": "Nouvelle liste",
+      "edit": "Éditer"
+    }
+  },
+  "login": {
+    "domain_name_label": "Tout d'abord, connectons-nous à votre instance Mastodon.  Entez votre nom de domaine:",
+    "search": "Rechercher",
+    "login": "Connexion"
+  },
+  "authorize": {
+    "manually_1": "La page d'autorisation est à présent affichée dans votre navigateur.",
+    "manually_2": "Si ce n'était pas le case, veuillez ouvrir le lien suivant manuellement.",
+    "code_label": "Recopier le code d'autorisation depuis votre navigateur:",
+    "submit": "Envoyer"
+  },
+  "receive_drop": {
+    "drop_message": "Déposer pour téléverser sur Mastodon"
+  },
+  "message": {
+    "account_load_error": "Erreur au chargement des comptes",
+    "account_remove_error": "Erreur à la suppression du compte",
+    "preferences_load_error": "Erreur au chargement des préférences",
+    "timeline_fetch_error": "Erreur au chargement du fil public",
+    "notification_fetch_error": "Erreur au chargement des notifications",
+    "favourite_fetch_error": "Erreur au chargement des favoris",
+    "start_streaming_error": "Erreur au démarrage du flux",
```

Comment:
__[Misspell]__

"marrage" is a misspelling of "marriage"


<!-- Comment by [Sider](https://sider.review) -->

Reply:
Same as above.  "Démarrage" = "Start"